app/src/main/java/com/example/hungrywolfes/ui/detailsScreen/DetailsFragment.kt

```diff
@@ -50,11 +45,14 @@ class DetailsFragment : Fragment() {
     }
 
     private fun getArgumentsFromHomeScreen() {
```

Comment:
Why exactly have you added two identical methods? 
Also..why are you still getting the argument 'manually'? Since you added safeArgs, you don't need to do anything else.